adaptavist/adaptavist.py

```diff
@@ -571,19 +571,21 @@ def create_test_run(self, project_key: str, test_run_name: str, **kwargs: Any) -
         test_plan_key: str = kwargs.pop("test_plan_key", "")
         test_cases: List[str] = kwargs.pop("test_cases", [])
         environment: str = kwargs.pop("environment", "")
+        version: str = kwargs.pop("version", "")
         raise_on_kwargs_not_empty(kwargs)
 
         self.create_folder(project_key=project_key, folder_type=TEST_RUN, folder_name=folder)
 
         test_cases_list_of_dicts = [{
             "testCaseKey": test_case_key,
-            "environment": environment or None,
+            "environment": environment or None
```

Comment:
```suggestion
            "environment": environment or None,
```

He always keep the comma to prevent git changes in case a dict needs to be extended.